src/poliastro/twobody/orbit.py

```diff
@@ -388,9 +388,12 @@ def from_body_ephem(cls, body, epoch=None):
                 "{}. Use Time(..., scale='tdb') instead.".format(epoch.tdb.value),
                 TimeScaleWarning,
             )
-
-        r, v = get_body_barycentric_posvel(body.name, epoch)
-
+        try:
+            r, v = get_body_barycentric_posvel(body.name, epoch)
+        except KeyError:
+            raise RuntimeError(
+                "To compute the semimajor axis or eccentricity for Moon and Pluto use the JPL ephemeris:\n>>>from astropy.coordinates import solar_system_ephemeris\n>>>solar_system_ephemeris.set('jpl')"
```

Comment:
```suggestion
                "To compute the position and velocity of the Moon and Pluto use the JPL ephemeris:\n>>>from astropy.coordinates import solar_system_ephemeris\n>>>solar_system_ephemeris.set('jpl')"
```

Reply:
On it! And my apologies for the previous pull request, I messed up the git rebase. Sorry for the inconvenience caused.